src/index.js

```diff
@@ -1,5 +1,5 @@
 import h from "./h"
 import app from "./app"
-import router from "./router"
+import Router from "./router"
```

Comment:
@jbucaran Generally I'm in favor of stuff like this. Especially now that the router is basically just an app/plugin I think we can go for this.

I don't think it's necessary yet though. Basically the decision is quite opinionated.

This won't really solve the examples problem though. People will still see that it's `import Router from 'hyperapp-router'`.